presto-docs/src/main/sphinx/admin/function-namespace-managers.rst

```diff
@@ -0,0 +1,88 @@
+===========================
+Function Namespace Managers
+===========================
+
+.. warning::
+
+    This is an experimental feature being actively developed. The way
+    Function Namespace Managers are configured might be changed.
+
+Function namespace managers support storing and retrieving SQL
+functions, allowing Presto engine to perform actions such as creating,
+altering, deleting, and executing functions.
```

Comment:
"executing functions" is not new and it's not something "allowed" by function namespace managers.

Reply:
I intend to say `resolving functions`

Reply:
resolving function is also not introduced by function namespace manager though.